tests/speculos/test_configuration_cmd.py

```diff
@@ -1,10 +1,3 @@
 
 def test_configuration(cmd):
-    if cmd.model == "nanos":
-        assert cmd.get_configuration() == (14, 1, 10, 4)
-
-    if cmd.model == "nanox":
-        assert cmd.get_configuration() == (14, 1, 10, 4)
-
-    if cmd.model == "nanosp":
-        assert cmd.get_configuration() == (14, 1, 10, 4)
+    assert cmd.get_configuration() == (2, 1, 10, 4)
```

Comment:
will have to be updated or removed at some point (avoid hardcoded values here)

Reply:
These tests will be ultimately replaced by Ragger tests :+1: